front_end/content.py

```diff
@@ -585,6 +585,36 @@ def get_registered_courses(self, user_id):
 
         return registered_courses
 
+    def get_partners_dict(self, course, user_id):
```

Comment:
This looks good, but please make the function name more consistent with other names that we have been using. We haven't typically been indicating the type of object that is returned. Maybe something like `get_partner_info`.